src/logger.js

```diff
@@ -0,0 +1,35 @@
+const
```

Comment:
I'm not too familiar with that approach, but in general it sounds like a good idea.